tests/functional/test_requests.py

```diff
@@ -567,7 +567,9 @@ def test_httpretty_provides_easy_access_to_querystrings_with_regexes():
     })
 
 
+from unittest import skip
```

Comment:
This doesn't work with Python 2.6, where `unittest` doesn't have `skip`. A possible workaround is to do this:

    try:
        from unittest import skip
    except ImportError:
        from unittest2 import skip

which will allow it to work with Python 2.6 if you install the `unittest2` library which provides a backport of the 2.7 unittest for earlier Pythons.

Reply:
I think it might be time to drop python 2.6 support, thoughts?

Reply:
I think we're still building for RHEL 6 (for the EPEL repository, which is basically Fedora packages built for RHEL), which is Python 2.6. https://koji.fedoraproject.org/koji/buildinfo?buildID=836707 is the build I did around the time I was poking this.